docs/migration/5_0.md

```diff
@@ -813,3 +813,10 @@ Due to i18next migration, certain set of keys have been migrated (from `_plural`
 - Added `RegisterComponentService` that accepts `UserRegisterFacade` through constructor injection. `RegisterComponentService` has only those methods from `UserRegisterFacade` that are needed for `RegisterComponent`.
 - Updated the constructor of `RegisterComponent` and replaced `UserRegisterFacade` with `RegisterComponentService`.
 
+## Breaking changes in cdc intrgration library
+
+- When `CX_CDC` is enabled, the native Login / Register and Forgot Password screens uses CDC JS SDK to perform user account creation and authentication.
+- Added `CDCRegisterComponentService` that extends `RegisterComponentService`. It invokes CDC JS SDK to register the user and overrides the `postRegisterMessage()` to suppress the success message on sucessful registration.
+- Added `CdcLoginFormComponentService` that extends `LoginFormComponentService` and invokes CDC JS SDK to login the user.
+- Added `CDCForgotPasswordComponentService` that extends `ForgotPasswordComponentService` and invokes CDC JS SDK to send the account reset email.
```

Comment:
Please also mention the constructor changes to the `feature-libs/user/profile/components/register/register-component.service.ts`

Reply:
sure Nik